spec/defines/pack_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'spec_helper'
+require 'helpers/fact_helper'
+
+describe "st2::pack" do
+
+  let(:title) { 'st2testpack' }
+  let(:facts) do
+    {
+    :operatingsystem => 'RedHat',
+    :osfamily => 'RedHat',
+    :operatingsystemmajrelease => '7',
+    }
+  end
+
+  it do
+    is_expected.to contain_file('/opt/stackstorm').with(
+      :ensure => 'directory',
+      :owner => 'root',
+      :group => 'root',
+      :mode => '0755'
+    )
+  end
+  it do
+    is_expected.to contain_file('/opt/stackstorm/configs').with(
+      :ensure => 'directory',
+      :owner => 'st2',
+      :group => 'root',
+      :mode => '0755'
+    )
+  end
+  it do
+    is_expected.to contain_file('/opt/stackstorm/packs').with(
+      :ensure => 'directory',
+      :owner => 'root',
+      :group => 'st2packs',
```

Comment:
@ruriky it looks like this trailing comma is what's causing you grief in the rspec testing for older versions

here are the other spots where travis found an issue:

```
spec/defines/pack_spec.rb:36: syntax error, unexpected ')'
spec/defines/pack_spec.rb:67: syntax error, unexpected ')'
spec/defines/pack_spec.rb:81: syntax error, unexpected ')'
spec/defines/pack_spec.rb:92: syntax error, unexpected ')'
spec/defines/pack_spec.rb:105: syntax error, unexpected ')'
```

should be an easy fix!